packages/iTwinUI-react/src/core/utils/components/VirtualScroll.tsx

```diff
@@ -35,7 +36,7 @@ const getElementHeight = (element: HTMLElement | undefined) => {
 
 const getElementHeightWithMargins = (element: HTMLElement | undefined) => {
   if (!element) {
-    return 0;
+    return undefined;
```

Comment:
why undefined?

Reply:
I thought it makes more sense. If element is not existing (i.e. there is single child and I am checking for the 2nd childs height) I should not get height value at all. And then check I had for `childHeights` is easier, to assign first childs height to all of the children.
I better return height `0` when element is existing but it is empty, for example.